Connectors/Rhino/Speckle.Connectors.GrasshopperShared/Components/Objects/CreateSpeckleProperties.cs

```diff
@@ -61,46 +54,23 @@ protected override void SolveInstance(IGH_DataAccess da)
       }
 
       properties.Add(inputName, new());
-
-      // create property for the branch of the input
-      for (int i = 0; i < branchCount; i++)
-      {
-        var data = inputParam.VolatileData.get_Branch(i).OfType<object>().ToList();
-        if (data.Count > 1)
-        {
-          AddRuntimeMessage(GH_RuntimeMessageLevel.Error, $"Cannot support list properties yet.");
-          return;
-        }
-
-        SpecklePropertyGoo propGoo = new();
-        if (!propGoo.CastFrom(data.Count == 0 ? null : data.First()))
-        {
-          AddRuntimeMessage(
-            GH_RuntimeMessageLevel.Error,
-            $"Parameter {inputParam.NickName} should not contain anything other than strings, doubles, ints, and bools."
-          );
-
-          return;
-        }
-
-        propGoo.Path = inputName;
-        properties[inputName].Add(propGoo);
-      }
     }
 
-    // now create the property groups
-    for (int i = 0; i < branchCount; i++)
+    for (int i = 0; i < Params.Input.Count; i++)
     {
-      var currentProps = new Dictionary<string, SpecklePropertyGoo>();
-      foreach (var kvp in properties)
+      object? value = null;
+      var success = da.GetData(i, ref value);
+      var actualValue = value?.GetType().GetProperty("Value").GetValue(value); // note: unsure if reflection here hurts our performance
+      if (!success || value == null || actualValue == null)
       {
-        currentProps[kvp.Key] = kvp.Value[i];
+        continue;
```

Comment:
We probably want to add a flag before continuing, so the user knows their props are a little borked: 
```AddRuntimeMessage(
            GH_RuntimeMessageLevel.Error,
            $"Parameter {inputParam.NickName} should not contain anything other than strings, doubles, ints, and bools."
          );```